spateo/plotting/static/pairwise.py

```diff
@@ -0,0 +1,185 @@
+                (-1.0, 1.0),
+                # (1., 1.),
+                (1.0, -1.0),
+                (-1.0, -1.0),
+            )
+        )
+
+        right_triangle = np.array(
+            (
+                (-1.0, 1.0),
+                (1.0, 1.0),
+                (1.0, -1.0),
+                # (-1., -1.)
+            )
+        )
+
+        polygon_list = []
+        color_list = []
+
+        ax.set_ylim(-0.55, label.num_labels - 0.45)
+        ax.set_xlim(-0.55, label.num_labels - 0.45)
+
+        for label_1 in range(connections.shape[0]):
+            for label_2 in range(connections.shape[1]):
+
+                if label_1 <= label_2:
+
+                    for triangle in [left_triangle, right_triangle]:
+                        center = np.array((label_1, label_2))[np.newaxis, :]
+                        scale_factor = connections[label_1, label_2] / connections_max
+                        offsets = triangle * max_scale * scale_factor
+                        polygon_list.append(center + offsets)
+
+                    color_list += (id_colors[label.ids[label_2]], id_colors[label.ids[label_1]])
+
+        collection = PolyCollection(polygon_list, facecolors=color_list, edgecolors="face", linewidths=0)
+
+        ax.add_collection(collection)
+
+        ax.tick_params(labelbottom=False, labeltop=True)
+        ax.xaxis.set_tick_params(pad=-2)
+    else:
+        # Heatmap of connection strengths
+        heatmap = ax.imshow(connections, cmap=colormap, interpolation="nearest")
+
+        divider = make_axes_locatable(ax)
+        cax = divider.append_axes("right", size="5%", pad=0.1)
+
+        fig.colorbar(heatmap, cax=cax)
+        cax.tick_params(axis="both", which="major", labelsize=6, rotation=-45)
+
+        # Change formatting if values too small
+        if connections_max < 0.001:
+            cax.yaxis.set_major_formatter(StrMethodFormatter("{x:,.1e}"))
+
+    # Formatting adjustments
+    ax.set_aspect("equal")
+
+    ax.set_xticks(
+        np.arange(label.num_labels),
+    )
+    # If label has categorical labels associated, use those to label the axes instead:
+    if label.str_map is not None:
+        ax.set_xticklabels(
+            label.str_ids,
+            fontsize=label_fontsize,
+            fontweight="bold",
+            rotation=90,
+            path_effects=[PathEffects.Stroke(linewidth=0.5, foreground="black", alpha=0.8)],
+        )
+    else:
+        ax.set_xticklabels(
+            label.ids,
+            fontsize=label_fontsize,
+            fontweight="bold",
+            rotation=0,
+            path_effects=[PathEffects.Stroke(linewidth=0.5, foreground="black", alpha=0.8)],
+        )
+
+    ax.set_yticks(np.arange(label.num_labels))
+    if label.str_map is not None:
+        ax.set_yticklabels(
+            label.str_ids,
+            fontsize=label_fontsize,
+            fontweight="bold",
+            path_effects=[PathEffects.Stroke(linewidth=0.5, foreground="black", alpha=0.8)],
```

Comment:
make the patheEffects stroke an argument also? when it is None, it uses the above default values but when it is set, it needs to be a list of `PathEffects.Stroke`